app/models/library_entry.rb

```diff
@@ -134,5 +143,12 @@ def activity
       user.update_feed_completed!
       user.update_profile_completed!
     end
+
+    # Sync MAL updates if linked profile exists
+    MyAnimeListSyncWorker.perform_async(self, 'create/update') if sync_to_mal?
```

Comment:
One final thing and then I think we can merge this — you don't wanna pass the whole object into a worker, it gets nasty and messy

Instead of `self`, pass `id`, and then in the worker, do `LibraryEntry.find(library_entry_id)`